data-raw/organizations/VIGO/prepare-VIGO.R

```diff
@@ -0,0 +1,55 @@
+# VIGO Preparation Script
+
+# This is a template for importing, cleaning, and exporting data
+# ready for many packages universe.
+
+# Stage one: Collecting data
+VIGO <- haven::read_dta("data-raw/organizations/VIGO/isq_2017.dta")
+
+# Stage two: Correcting data
+# In this stage you will want to correct the variable names and
+# formats of the 'VIGO' object until the object created
+# below (in stage three) passes all the tests.
+# We recommend that you avoid using one letter variable names to keep
+# away from issues with ambiguous names down the road.
+VIGO <- as_tibble(VIGO) %>%
+  manydata::transmutate(igoID = reo,
+                        Label = manypkgs::standardise_titles(reo),
+                        Year = as.character(year)) %>%
+  dplyr::group_by(igoID) %>%
+  dplyr::mutate(Beg = min(Year)) %>%
+  dplyr::relocate(igoID, Label, Beg, Year, vitality, zombie, life, dead) %>%
```

Comment:
What is the relationship between 'Beg' and 'year'? Is 'Beg' just 'Year'? Or are there also end dates in there somewhere?

Reply:
There are no end dates, `Beg` codes the founding date for the IGO whereas `Year` captures the specific year for which the IGO is scored in the `vitality`, `zombie`, `life`, or `dead` indicators. I kept the `Year` variable because the scores change across years.